src/components/ClauseComponent.js

```diff
@@ -154,7 +154,7 @@ function ClauseComponent(props) {
     {
       !props.readOnly
       && Object.entries(conditionals).map(([key, value]) => (
-        value.isFalse && (value.whenFalse === '')
+        value.isFalse
```

Comment:
The addition symbol is showing up when there is a `whenFalse` value that isn't `''`, so I think this line may need to be reverted.

Reply:
Shouldn't this be the case? Otherwise how would someone revert back to the ```whenTrue``` value? I am a bit confused.

Reply:
We want the `ConditionalBoolean` (the addition symbol) to _only_ render when the value of the conditional variable is false **AND** that false value is an empty string.

Reply:
Otherwise, we want the `ConditionalBoolean` to **NOT** render, and we want the `ConditionalPlugin` to handle the render because it's a value we want to display, be it a true value which will be toggled to an empty string (removed) or toggled to a false value which is just _some other string_

Reply:
Oh okay, got it then. I will revert this. Thank you for the clarification!

Reply:
@irmerk fixed it! 
If this is okay, I'll squash and remove the 'something' from the ```whenFalse```

Reply:
Sounds good. Can you actually copy paste in another conditional somewhere with `''` instead of `'something'` so we can be absolutely sure this works for both types of conditionals?